tests/js/spec/views/settings/organizationMembers/organizationMembersList.spec.jsx

```diff
@@ -154,11 +159,14 @@ describe('OrganizationMembersList', function () {
       TestStubs.routerContext([{organization}])
     );
 
-    wrapper.find('Button[data-test-id="remove"]').at(0).simulate('click');
+    await act(async () => {
```

Comment:
do we need `act` here?

Reply:
Probably not, unless it would scream of unwrapped act update, let me test it